operators/src/util/raster_stream_to_geotiff.rs

```diff
@@ -196,7 +196,9 @@ where
         if let Some(no_data) = gdal_tiff_metadata.no_data_value {
             band.set_no_data_value(Some(no_data))?;
         } else {
-            band.create_mask_band(false)?;
+            // only allowed option for internal masks
+            band.create_mask_band(true)?;
```

Comment:
I don't understand this but I believe you

Reply:
Says GDAL :shrug: